packages/components/forms/src/checkbox/Checkbox.tsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
+import { Flex } from '@contentful/f36-core';
 import { BaseCheckbox } from '../base-checkbox';
 import type { CheckboxProps as _CheckboxProps } from '../base-checkbox/types';
+import { Label } from '../Label';
+import { getStyles } from './Checkbox.styles';
 
 export type CheckboxProps = _CheckboxProps;
 
-export const _Checkbox = (
-  { testId = 'cf-ui-checkbox', ...otherProps }: CheckboxProps,
-  ref: React.Ref<HTMLLabelElement>,
+const _Checkbox = (
+  { testId = 'cf-ui-checkbox', label, id, className, ...otherProps },
+  ref: React.Ref<HTMLDivElement>,
 ) => {
+  const styles = getStyles();
```

Comment:
Since this will return the same thing every time
Wdyt about calling it outside of the component?

Reply:
we don't want to create styles before the component is rendered, that's why we can them here. this pattern is consistent across all components.

Reply:
Also as emotion works, it stores all the classNames, so it would return the memoized className

Reply:
so we are using emotion wrong in user_interface?

Reply:
yeah, we could use it better in user_interface